PriscillaSam/Stackoverflow-Lite
# #163005196 Implement functionality to accept an answer

#### What does this PR do?
Implement functionality to accept an answer

#### Description of Tasks to be completed
- update profile reducer to handle post question action
- update tests based on new changes

#### How can this be manually tested?
- on the terminal run `git clone https://github.com/PriscillaSam/Stackoverflow-Lite.git`.
- run `git checkout ft-accept-answer-react-163005196`.
- run `npm install` to install the npm packages.
- run `npm run start:dev` to start the server.
- sign up and navigate to localhost:3000/questions on your browser to post a question.
- post an answer to the question. The accept button should come up on the answer

#### Screenshots where applicable
<img width="1292" alt="screen shot 2019-01-15 at 12 54 10 pm" src="https://user-images.githubusercontent.com/29674347/51179029-b1f83080-18c4-11e9-82f9-1bd10b89fd9b.png">

#### What are the relevant Pivotal Tracker stories?
[#163005196](https://www.pivotaltracker.com/story/show/163005196)
